packages/docusaurus/src/webpack/__tests__/utils.test.ts

```diff
@@ -83,63 +76,14 @@ describe('extending generated webpack config', () => {
       },
     });
     const errors = validate(config);
-    expect(errors.length).toBe(0);
-  });
-
-  test('webpack-merge with custom strategy', async () => {
```

Comment:
This is now handled within webpack merge, and I really don't have the mental energy to rewrite this test

Reply:
thanks, will take a look at this